### ISSUE TITLE
When using `onlySpecified` it complains about .bin

### ISSUE THREAD

**Issue by Page-** [mzgol/grunt-check-dependencies#7]
```
checkDependencies: {
	this: {
		options: {
			onlySpecified: true
		}
	}
}
```

```
grunt checkDependencies
Running "checkDependencies:this" (checkDependencies) task
>> Package .bin installed, though it shouldn't be
```

**Comment by mzgol** [comment]
Yeah, I didn't test it on a project that has binaries in dependencies... Thanks for the report!

**Comment by mzgol** [comment]
Fix is included in 0.9.0.

**Comment by Page-** [comment]
Thanks :)
